Lesson12/Lesson12_1.cpp

```diff
@@ -0,0 +1,42 @@
+// Lesson 12 1
+#include <iostream>
+
+void bubbleSort(int arr[], int size){
+    int tmp = 0;
+
+    for(int n = 0; n < size - 1; n++){
+        for(int i = 0; i < size - 1 - n; i++)
+        {
+            if(arr[i] > arr[i + 1]){
+                tmp = arr[i];
+                arr[i] = arr[i + 1];
+                arr[i + 1] = tmp;
+            }
+        }
+    }
+
+    std::cout << "Bubble sorted array: "<< std::endl;
+    for(int i = 0; i < size; i++){
+        std::cout << "[" << arr[i] << "]";
+    }
+    std::cout << std::endl;
```

Comment:
you should print it outside this function

Reply:
fixed